Remove requirement to fetch git tag for version info
as the source tarball isn't a git checkout.

author: micketeer@gmail.com
--- meson.build.orig	2025-12-15 10:56:49.665763580 +0100
+++ meson.build	2025-12-15 10:56:55.756674543 +0100
@@ -196,7 +196,7 @@
 add_project_arguments(feature_cflags, language: 'c')
 
 xfce_revision_h = vcs_tag(
-  command: ['git', 'rev-parse', '--short', 'HEAD'],
+  # command: ['git', 'rev-parse', '--short', 'HEAD'],
   fallback: 'UNKNOWN',
   input: 'xfce-revision.h.in',
   output: 'xfce-revision.h',
